app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package io.pumpkinz.pumpkinreader.model;
+
+import java.util.Date;
+
+
+public abstract class Item {
```

Comment:
Or maybe Item could inherit ItemPOJO?

Also BTW, what is POJO naming best practice?
I think it's the same with model naming convention, as they are POJO alike.
But CMIIW.